src/cbmc/cbmc_parse_options.cpp

```diff
@@ -994,6 +1001,8 @@ void cbmc_parse_optionst::help()
     " --object-bits n              number of bits used for object addresses\n"
     " --dimacs                     generate CNF in DIMACS format\n"
     " --beautify                   beautify the counterexample (greedy heuristic)\n" // NOLINT(*)
+    " --random-seed %f             SAT checker randomizer seed\n" // NOLINT(*)
```

Comment:
Seed should probably be treated as an opaque 64 bit bit-vector.  Minisat's use of it as a double is just implementational convenience.  If we use it to seed other random number generators such as srand() (which is probably a good idea) they will want it as an integer.